packages/@lwc/engine/src/framework/wiring.ts

```diff
@@ -102,7 +101,7 @@ function createContextWatcher(
                 callbackWhenContextIsReady(newContext);
             },
         });
-        dispatchEvent.call(elm, internalDomEvent);
+        elm.dispatchEvent(internalDomEvent);
```

Comment:
Should we wait to make this change until we're ready to refactor the way wire contexts are resolved? It seems risky to assume that the dispatchEvent() method hasn't been or won't be redefined by components.

Reply:
Sure, let me pull this one out.